apps/req.c

```diff
@@ -495,6 +505,10 @@ int req_main(int argc, char **argv)
     if (!gen_x509) {
         if (days != UNSET_DAYS)
             BIO_printf(bio_err, "Ignoring -days without -x509; not generating a certificate\n");
+        if (not_before != NULL)
+            BIO_printf(bio_err, "Ignoring -not_before without -x509; not generating a certificate\n");
+        if (not_after != NULL)
+            BIO_printf(bio_err, "Ignoring -not_after without -x509; not generating a certificate\n");
```

Comment:
Nice - please add such a warning also for `-days`.

And please prepend to all such messages: `Warning: `

Reply:
See lines 823-824:
```c
if (not_after != NULL && days != UNSET_DAYS)                                                                                                                                                                                
    BIO_printf(bio_err, "Warning: -not_after option overriding -days option\n");
```